### PR TITLE
Issue #1566: HideUtilityClassConstructor violations fixed

### PR DESCRIPTION
**Pull request by rdiachenko**
Violations fixed:
- HideUtilityClassConstructor - Utility classes should not have a public or default constructor.

### CODE REVIEW

File: src/main/java/com/puppycrawl/tools/checkstyle/gui/Main.java [hunk 1]
```diff
@@ -37,6 +37,13 @@
     private static JFrame frame;
 
     /**
+     * Hidden constructor of the current utility class.
+     */
+    private Main() {
+        throw new IllegalStateException("Utility class constructor mustn't be called.");
```
Comment (mkordas) [inline src/main/java/com/puppycrawl/tools/checkstyle/gui/Main.java]: I don't think that throwing an exception is necessary. In majority of other places I think we have just empty private constructor.

Comment (romani) [inline src/main/java/com/puppycrawl/tools/checkstyle/gui/Main.java]: agree with mkordas

**Comment by rdiachenko** [comment]
"throw ..Exception" removed

**Comment by romani** [comment]
merged as FF

**Pull request closed without merging**